src/hooks/useDailyNotes.ts

```
@@ -169,6 +184,7 @@ export const useDailyNotes = (selectedDate?: Date) => {
     title: string;
     content: string;
     tags: string[];
+    references?: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)